protocol/river-status-unstable-v1.xml

```diff
@@ -75,9 +75,18 @@
       </description>
       <arg name="tags" type="array" summary="array of 32-bit bitfields"/>
     </event>
+
+    <event name="urgent_tags" since="2">
+      <description summary="urgent tags of the output">
+        Sent once binding the interface and again whenever a tag contain a
+        view marked as urgent or when losing its urgency. Tags stay till
+        the view loses its urgency.
+      </description>
+      <arg name="tags" type="uint" summary="32-bit bitfield"/>
+    </event>
   </interface>
 
-  <interface name="zriver_seat_status_v1" version="1">
+  <interface name="zriver_seat_status_v1" version="2">
```

Comment:
We actually don't need to bump this interface's version as neither it nor any of its child interfaces have been modified. See: https://wayland.freedesktop.org/docs/html/ch04.html#sect-Protocol-Versioning
```suggestion
  <interface name="zriver_seat_status_v1" version="1">
```